src/style.css

```diff
@@ -0,0 +1,29 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
style margins and paddings for each element you need, don't use `*`-selector for that